src/Simulation/Simulators.Tests/TestProjects/IntrinsicTests/IntrinsicTests.csproj

```diff
@@ -3,15 +3,13 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>false</IsPackable>
-    <ExposeReferencesViaTestNames>true</ExposeReferencesViaTestNames>
```

Comment:
Let me double check... I thought I had kept those tests around but it's possible I took them out by mistake.

Reply:
I did double check, and the tests that actually relied on this are still present and compile out of src\Simulation\Simulators.Tests\TestProjects\UnitTests\UnitTests.csproj, which still has `ExposeViaTestReferences` configured. I think this may have been copied in here accidentally and there wasn't any dependency on it, so I took it out.